cmd/worker/worker.go

```diff
@@ -354,6 +360,20 @@ func (c *Command) addClickhouseColumns(ctx context.Context, fields []*tunnel.Eve
 	return nil
 }
 
+func (c *Command) isEventsTableEmpty(ctx context.Context) (bool, error) {
+	var columns uint64
+	if err := c.clickhouse.QueryRow(ctx, `
+		SELECT COUNT(*)
+		FROM INFORMATION_SCHEMA.COLUMNS
+		WHERE (table_name = 'events') AND (table_schema = 'subtrace')
```

Comment:
Not sure there's a better way to get the number of columns in a Clickhouse table, I struggled for a bit before I settled on this. `DESCRIBE subtrace.events` doesn't seem to give a result that I can actually count the number of rows. See https://github.com/ClickHouse/ClickHouse/issues/8854 for more details.

Reply:
`SELECT COUNT(*) FROM system.columns` feels more appropriate. The `information_schema.columns` table seems to have way too many database internals. Plus there's precedent for using `system.columns` in this file; see `hasColumn`.